### ISSUE TITLE
Exception after upgrade from 3.1.2 to 3.1.4

### ISSUE THREAD

**Issue by somehan** [mathjax/MathJax#2683]
### Issue Summary

We're using mathjax in a Node backend.
After upgrading to 3.1.4, we get an exception in the mathjax library (cf. below for details).

### Steps to Reproduce:

1. Use mathjax 3.1.2 in node project, see it work
2. Upgrade to 3.1.4, get an error
3. Downgrade to 3.1.2 and see it work again
(4. Try out version 3.1.3 and see it fail as well)

### Technical details:

* Node version: 16.1.0
* Typescript version: 4.2.4
* MathJax Version: 3.1.4 
* Client OS: Debian Linux

### Supporting information:

* Here's how we import and call mathjax:
```
import * as MathJax from "mathjax";

const mathJax = await MathJax.init({
        loader: { load: ["input/tex", "output/svg"] },
    });
```

* Error:
```
TypeError: Cannot set property 'init' of undefined
    at /home/name/theFolder/code/alma/api/node_modules/mathjax/es5/node-main.js:1:254572
    at Object.<anonymous> (/home/name/theFolder/code/alma/api/node_modules/mathjax/es5/node-main.js:1:254702)
    at Module._compile (node:internal/modules/cjs/loader:1109:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1138:10)
    at Module.load (node:internal/modules/cjs/loader:989:32)
    at Function.Module._load (node:internal/modules/cjs/loader:829:14)
    at Module.require (node:internal/modules/cjs/loader:1013:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (/home/name/theFolder/code/alma/api/src/clients/contentful/tex-formula-renderer.ts:1:1)
```

**Comment by dpvc** [comment]
We upgraded to webpack 5 during the 3.1.4 release, and it appears that there are some differences that have affected this and that the `webpack.config.js` for `node-main` have to be adjusted.  I will make a PR for that, but in the meantime, can you use `mathjax-full` rather than just `mathjax` for now?  That will end up loading MathJax from the compiled `.js` files rather than the webpacked versions, which will avoid the error.

**Comment by somehan** [comment]
Thanks for the quick reply and proposal :+1: 

Unfortunately the proposed fix does not work for us with `mathjax-full` 3.1.4
Neither when running our express backend via `ts-node` in dev mode, nor compiling it and running via node works for us.

We'll be waiting for the `mathjax` update.

**Comment by dpvc** [comment]
> Unfortunately the proposed fix does not work for us with `mathjax-full` 3.1.4

Did you change the import to 

``` javascript
import * as MathJax from "mathjax-full";
```

as well?

If so, is the error the same as you had originally?  Can you try

``` javascript
import * as MathJax from "mathjax-full/components/src/node-main/node-main.js";
```

and see if that works?

**Comment by somehan** [comment]
We did try the first option and I will try the second one as well to answer your question.

Maybe not today, though.

**Comment by somehan** [comment]
Here is some sample code: https://github.com/somehan/mathjax-3.1.4-issue that shows the problem.
It's using `require(mathjax)` to avoid issues with `import()`.

**Comment by somehan** [comment]
Fixed in v3.2.0. Thank you! :pray:
